policybot/deploy/kube/templates/cron-flakechaser.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1beta1
```

Comment:
These periodic syncs are just for events that the bot missed in its subscription?

Reply:
Correct.

And it's a shame that GitHub doesn't provide a better way to do these. A few of the data types support "list anything changed after date XXX" which works great, but other data types require that I do a complete enumeration and manual one-by-one inspection. This can take forever due to rate limits, and often fails part way through due to unspecified errors returned  by  GH...